bin/environmentd-debug-zip

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+#
+# materialized — build and run a core service or test.
+
+if [ -z "${MZDEV_POSTGRES}" ]; then
+    POSTGRES='postgres://root@localhost:26257/materialize'
+else
+    POSTGRES=${MZDEV_POSTGRES}
+fi
```

Comment:
```suggestion
postgres=${MZDEV_POSTGRES:-postgres://root@localhost:26257/materialize}
```

Half of this is the succinct syntax for default values if a var isn't set; the other half is that lowercase variable names for non-environment variables are conventional.

Reply:
Done!